File: docs/basic/code/simulate/simulate_1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int u = input.nextInt();
        int d = input.nextInt();
        int n = input.nextInt();
        int time = 0, dist = 0;
        while (true) {  // Dùng vòng lặp vô hạn để liệt kê.
            dist += u;
            time++;
            if (dist >= n) {
                break;  // Thoát vòng lặp vô hạn khi thỏa điều kiện.
            }
            dist -= d;
        }
        System.out.println(time);   // In kết quả thu được.
        input.close();
    }
}

File: docs/basic/code/simulate/simulate_1.py
u, d, n = map(int, input().split())
time = dist = 0
while True:  # Dùng vòng lặp vô hạn để liệt kê.
    dist += u
    time += 1
    if dist >= n:  # Thoát vòng lặp vô hạn khi thỏa điều kiện.
        break
    dist -= d
print(time)  # In kết quả thu được.
